client/src/App.js

```diff
@@ -3,15 +3,18 @@ import { MuiThemeProvider } from "@material-ui/core";
 import { BrowserRouter, Route } from "react-router-dom";
 
 import { theme } from "./themes/theme";
-import LandingPage from "./pages/Landing";
+
 
 import "./App.css";
+import CreateForm from "./pages/Create";
+import EditForm from "./pages/Edit"
 
 function App() {
   return (
     <MuiThemeProvider theme={theme}>
       <BrowserRouter>
-        <Route path="/" component={LandingPage} />
+        <Route exact path="/" component={CreateForm} />
```

Comment:
This should probably be in a /create path or something like that, since the 'landing page' will be at '/'.

Reply:
Good catch!

I put this as '/' for the ease of viewing this during launch; but when merged with all branches, I'll correct the path :)